### DISCUSSION TITLE
Why `h-fit` doesn't set height to element?

### DISCUSSION THREAD

**Discussion by MonaAghili** [tailwindlabs/tailwindcss#14381] [Help]
I have a carousel where the wrapper has the `relative` class, and I've also added `h-fit`, but it seems like `h-fit` isn't applying, so the indicator wrapper isn't showing on the carousel. Is there any other height-related class I can use instead of `h-fit`?

https://play.tailwindcss.com/7kPsJIjo5S

**Comment by topherfangio** [comment]
You're applying absolute positioning to **all** of the images inside the container, so the container has no items and thus has no height.

Instead of `[&_.item]:absolute` try `[&_.item:not(.active)]:absolute`.

This way, the active item will always set the height of the container, and any non-active items will be absolute-positioned.

**Reply by MonaAghili** [reply]
Thank you so much for your suggestion! I really appreciate your help, and I'll definitely try it out.
